bento_beacon/app.py

```diff
@@ -57,12 +59,41 @@
 }
 
 with app.app_context():
+    # load blueprints
     endpoint_sets = current_app.config["BEACON_CONFIG"].get("endpointSets")
     for endpoint_set in endpoint_sets:
         if endpoint_set not in BEACON_MODELS:
             raise APIException(message="beacon config contains unknown endpoint set")
         app.register_blueprint(blueprints[endpoint_set])
 
+    # get censorship settings from katsu
+    max_filters = None
+    count_threshold = None
+    retries = 0
+    max_retries = current_app.config["MAX_RETRIES_FOR_CENSORSHIP_PARAMS"]
+    for tries in range(max_retries+1):
+        current_app.logger.info("calling katsu for censorship parameters")
+        try:
+            max_filters, count_threshold = katsu_censorship_settings()
+        except APIException:
+            # katsu down or unavailable, details logged when exception thrown
+            # swallow exception and continue retries
+            current_app.logger.error("error calling katsu for censorship settings")
+        
+        if max_filters is not None and count_threshold is not None:
+            break
+        sleep(5 + 5*tries)
+
+    # either params retrieved or we hit max retries
+    if max_filters is None or count_threshold is None:
+        # kill container
+        raise RuntimeError("unable to retrieve censorship settings from katsu")
+    else:
```

Comment:
don't need else here, since raise will always break out of the if block

Reply:
yes, this is here for emphasis only, perhaps the emphasis isn't needed.

Reply:
removed